src/library/Cardano/CoinSelection.hs

```diff
@@ -3,44 +3,55 @@
 {-# LANGUAGE DerivingVia #-}
 {-# LANGUAGE NumericUnderscores #-}
 {-# LANGUAGE RankNTypes #-}
+{-# OPTIONS_HADDOCK prune #-}
 
 -- |
 -- Copyright: © 2018-2020 IOHK
 -- License: Apache-2.0
 --
 -- Provides general functions and types relating to coin selection.
 --
--- The 'CoinSelectionAlgorithm' type provides a /common interface/ to coin
--- selection algorithms.
+-- The 'CoinSelection' type represents a __coin selection__, the basis for a
```

Comment:
If this is going on Haddock I think it's worth noting that this is for UTxO-based blockchains such as Bitcoin and Cardano.

Reply:
> If this is going on Haddock I think it's worth noting that this is for UTxO-based blockchains such as Bitcoin and Cardano.

Good point! Fixed in ea8568fba6669cb5e64a3fb17f1966a7d8fa7280.